pkg/noderesourcetopology/filter.go

```diff
@@ -181,3 +207,22 @@ func resourceFoundOnNode(resName v1.ResourceName, wantQuantity resource.Quantity
 	}
 	return false
 }
+
+func allocateFromNUMA(nodes NUMANodeList, numaID int, container v1.Container) {
+	var nRes v1.ResourceList
+	// find the correct NUMA ID's resources
```

Comment:
we have
```
type NUMANodeList []NUMANode
```
can we fix (perhaps in a later PR?) createNUMANodeList to guarantee that the `slice index == numaID`, so we can always safely write
```
nRes = nodes[numaID].Resources
```

Reply:
Yes. That can be a good idea, but let's do that on later PR as you mentioned.